torch/cuda/tunable.py

```diff
@@ -444,66 +444,66 @@ def _create_matrices(
     lda: int,
     ldb: int,
     ldc: int,
-    transA: bool,
-    transB: bool,
-    dtypeA: torch.dtype,
+    trans_a: bool,
```

Comment:
Please move any code change to a separate PR.
And for this one, not sure how ok it is to actually change the function argument names!